extensions/ql-vscode/src/view/common/CodePaths/CodePaths.tsx

```diff
@@ -40,6 +52,9 @@ export const CodePaths = ({
   return (
     <>
       <ShowPathsLink onClick={onShowPathsClick}>Show paths</ShowPathsLink>
+      <Label data-testid="shortest-path-length">
```

Comment:
Minor suggestion. I don't think we've generally used test ids in the codebase so it would be good to avoid it here. In the test, can we just look for a `span` with the matching text instead?

Reply:
👍 
Somehow I thought I couldn't make it work without the `test-id` due to the whitespaces 🙈 . - But it works now 😁